app/react-components/Project.js

```diff
@@ -395,7 +396,29 @@ const Project = ({
                 { key: 'num_mutations', title: '# Mutations'},
                 {
                   title: <i className="fa fa-bar-chart-o"><div style={styles.hidden}>add to survival plot</div></i>,
-                  onClick: (d) => setSurvivalGene(d === survivalGene ? null : d),
+                  onClick: (d) => {
+                    if (d.survivalId !== geneData.id) {
+                      const filters = [
+                        // TODO: switch to genes.symbol filters when api is ready.
```

Comment:
Was just waiting for the data to be there so I could test the queries. Does the elastic 5 update include the data needed?